api/RTCIceCandidate.json

```
@@ -68,10 +68,12 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "Beginning in Firefox 68, the constructor's <code>options</code> parameter is optional."
```

Presumably as the optionality of the parameter becomes the norm, this note will become less salient. What do you think of framing this more historically, like this?

```suggestion
              "notes": "Before Firefox 68, the constructor's <code>options</code> parameter is required."
```